scipy/special/_support_alternative_backends.py

```diff
@@ -161,11 +227,14 @@ def __stdtr(df, t):
 
 
 def _stdtrit(xp, spx):
-    betainc = _get_native_func(xp, spx, 'betainc')  # noqa: F811
+    betainc = _get_native_func(spx, 'betainc')
+    stdtr = _get_native_func(spx, 'stdtr')
+    if stdtr is None:
+        stdtr = _stdtr(xp, spx)
```

Comment:
This is just defensive programming. 
All backends that implement betainc also happen to implement stdtr.